python-spec/testing/test_collection.py

```diff
@@ -19,3 +21,12 @@ def test_basic(self):
         md["hello"] = "world"
 
         self.assertEqual("world", coll.metadata["hello"])
+
+    def test_mro(self):
```

Comment:
please spell out (as the function name) the abbreviation `mro`

Reply:
Since [the term `mro` is used in the Python object model](https://docs.python.org/3/library/stdtypes.html#class.__mro__), I think it makes sense to use here while providing its expansion in the comment.

Reply:
Please spell out `test_method_resolution_order`

Ours is a package for users with varying strengths, skilled in different languages

While Python-expert software developers may have memorized certain insider acronyms, let's go ahead and take a few keystrokese to communicate clearly to everyone else.

Reply:
Given that this is test code, I'm not hugely attached to either; I stuck the MRO acronym into the comment so it is still visible.